endurance-page-cache.php

```diff
@@ -1329,6 +1329,11 @@ public function toggle_nginx( $new_value = 0 ) {
 				if ( ! is_dir( $path . '.cpanel/proxy_conf' ) ) {
 					mkdir( $path . '.cpanel/proxy_conf' );
 				}
+				$cf_enabled = get_option( 'endurance_cloudflare_enabled', false );
+
+				if ( $cf_enabled == true ) {
```

Comment:
```suggestion
				if ( $cf_enabled === true ) {
```

Reply:
I thought about that but was worried that depending on how that value gets set I think it will come back out of the db as either 'true' or 1. Would it be better to cast $cf_enabled to boolean and then do the triple strict comparison?

Reply:
That sounds like a good option.